cmd/kind/get/kubeconfig/kubeconfig.go

```diff
@@ -58,6 +58,15 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *flagpole) error {
+	// Check if the cluster name exists
+	known, err := cluster.IsKnown(flags.Name)
```

Comment:
should we check is known?

Reply:
no, we should try to get the kubeconfig and fail if we can't find a data source. the fact that maybe a loadbalancer or worker container still exists is not helpful